### ISSUE TITLE
Security vulnerability and new Toxcore release

### ISSUE THREAD

**Issue by Zigsaw** [isotoxin/isotoxin#20]
https://blog.tox.chat/2018/04/security-vulnerability-and-new-toxcore-release/

Please update toxcore to 0.2.2 if you haven't yet. (The last release uses toxcore 0.2.0 and by looking at commits with the keyword "toxcore", it doesn't seem you have)

**Comment by arfgh** [comment]
yes, it should be interesting that updating because isotoxin actually with other clients such as utox or qtox, is having problems, with audio chat, and other issues... This client is amazing, and the lacking of that old toxcore 0.2.0, begin to be crticial :(

**Comment by zoff99** [comment]
please be careful with old versions of toxcore:

https://blog.tox.chat/2021/12/stack-based-buffer-overflow-vulnerability-in-udp-packet-handling-in-toxcore-cve-2021-44847/

We urge everyone to update to [Toxcore 0.2.13](https://github.com/TokTok/c-toxcore/releases/tag/v0.2.13) as soon as possible.

**Comment by arfgh** [comment]
qtox is using just now 0.2.15 and uTox 0.2.13...
i am guessing why qTox that seems the most updated client, is using the 0.2.15 and not the current version that is 0.2.18....
